{"Launchpad":"https://bugs.launchpad.net/bugs/1452002","Id":"1452002","Gerrit":"https://review.openstack.org/180723","IdGerrit":"180723","IdCommit":"dabbaeda979dd441fc8376a63ec6d5d232656eff","Files":["cinder/tests/test_hds_hnas_backend.py","cinder/volume/drivers/hds/hnas_backend.py"],"Lines_Inseted":1,"Lines_Deleted":1,"CommitParent":"a5908fdbcb2b80fdd7cecc04c82abf7c88d1a75e","Classification":"Bug","KeywordsTitle":"driver fails, unhealthy","Description":"removes, unhealthy,fails, ERROR, ","KeywordsCommit":"fails, patch fixes, error, unhelathy,","Comments":"It's a bug report, a driver fails after removing a file system unhelathy."}